kerl

```diff
@@ -894,6 +899,17 @@ endif
 
 rehash
 ACTIVATE_CSH
+
+    otpver=$(get_otp_version "$rel")
+    if [ $otpver -ge 20 ]; then
+        # https://twitter.com/mononcqc/status/877544929496629248
+        cat << OTP_20_ADDITION >> "$absdir/activate"
+_KERL_SAVED_ERL_AFLAGS="\$ERL_AFLAGS"
+export _KERL_SAVED_ERL_AFLAGS
+export ERL_AFLAGS="-kernel shell_history enabled \$ERL_AFLAGS"
```

Comment:
We should wrap this with a check for `-kernel shell_history` not already being present. Something along the lines of the following I guess:

```
if [[ "\$ERL_AFLAGS" != *-kernel\ shell_history* ]]; then
    export ERL_AFLAGS="-kernel shell_history enabled \$ERL_AFLAGS"
fi
```

Need to verify that glob match is portable enough, though.